src/main/java/com/gooddata/md/maintenance/PartialMdArtifact.java

```diff
@@ -16,6 +12,7 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 class PartialMdArtifact {
 
+    @JsonProperty("status")
```

Comment:
our contribution rules requires to annotate getter, so what about to create package private getter? @martiner what do you think?